whois-api/src/main/java/net/ripe/db/whois/api/rest/WhoisRestService.java

```diff
@@ -231,6 +241,8 @@ public Response create(
             final RpslObject submittedObject = getSubmittedObject(request, resource, isQueryParamSet(unformatted));
             validateSubmittedCreateObject(request, submittedObject, objectType);
 
+            addBasic(request, passwords);
+
```

Comment:
is it only for this specific WhoisRestService or any http request through port 43? If I check WhoisServletDeployer there are lot more controllers that uses passwords as query parameter.

Reply:
I agree you should also add to Syncupdates